lib/vsvector.cpp

```diff
@@ -844,7 +844,7 @@ thread_local double new_maxlen;
 void VisualizationSceneVector::DrawVector(double px, double py, double vx,
                                           double vy, double cval)
 {
-   double zc = 0.5*(bb.z[0]+bb.z[1]);
+   double zc = (drawvector > 3)?(bb.z[1]):(0.5*(bb.z[0]+bb.z[1]));
```

Comment:
Instead of additional options for `v`, maybe it is better to add an input option `lambda` (by default `0.5`) that can adjust the `zc` value to `(1-lambda)*bb.z[0]+ lambda*bb.z[1]`?

This is similar to how we handle e.g. level lines with <kbd>F5</kbd> or rules positions with <kbd>~</kbd>

Reply:
I do not know, I do not see much reason for an arbitrary position, I see only a reason for mid-plane and above. The second thing is that color. The other options (except the first one) use color coding, so the arrows are almost invisible (blended) when looking from above.